src/DocumentFragment.js

```diff
@@ -0,0 +1,38 @@
+  // ### DocumentFragment Shim
+  //
+  // A representation of a document fragment that can be normally generated
+  // by calling `document.createDocumentFragment`. Since the document fragment
+  // is itself a node, it inherits all properties of a node.
+module.exports = (function(){
+  "use strict";
+
+  var Node = require('./Node');
+
+  function DocumentFragment() {
+    Node.call(this, Node.DOCUMENT_FRAGMENT_NODE, "#document-fragment");
+  }
+  DocumentFragment.prototype = Object.create(Node.prototype);
```

Comment:
I would advocate sidestepping Node entirely, and instead setting up a "fresh" prototype using
```
Document.prototype = {
  fname1: function(...) {
    ...
  },
  fname2: ...
  ...
}
```
with all the functionality we need from Node simply added anew. This should be very little code.